tests/ui/imports/suggest-remove-issue-121315.rs

```diff
@@ -0,0 +1,29 @@
+//@ run-rustfix
+//@ compile-flags: --edition 2021
+#![deny(unused_imports)]
+#![allow(dead_code)]
+
+// Test remove FlatUnused
+use std::convert::TryFrom;
+//~^ ERROR the item `TryFrom` is imported redundantly
+
+// Test remove NestedFullUnused
+use std::convert::{TryInto};
+//~^ ERROR the item `TryInto` is imported redundantly
+
+// Test remove NestedPartialUnused
+use std::convert::{From, Infallible};
+//~^ ERROR the item `From` is imported redundantly
+
+// Test remove both redundant and unused?
+// use std::convert::{AsMut, Into};
```

Comment:
The problem now is here, `AsMut` is an unused_import, `Into` is redundant_import, because we suggest removing at different places, there will be an error in rustfix: 

```
rustfix: Could not replace range 499...505 in file -- maybe parts of it were already replaced?
```

I will try to merge these different kinds of removing, so that `check_for_redundant_imports` will handle well for removing spans.

Reply:
@petrochenkov @surechen 
do you think is it a good direction to go?

Reply:
Done with commit https://github.com/rust-lang/rust/pull/121958/commits/300c184d2cf0139da2e54e6da4baa5432dd51f48

There are some test cases are failing, I need more cleanup on the labels.

How to show the labels if there are `AsMut` is `unused_import` and `Into` is `redundant_import `, they are all need to be removed: 
```rust
 use std::convert::{AsMut, Into};
```

😂